package.json

```
@@ -18,15 +18,17 @@
     "build": "webpack",
     "format": "prettier \"**/*.{css,js,ts,tsx,md}\" --write",
     "format:check": "prettier \"**/*.{css,js,ts,tsx,md}\" --check",
-    "prepare": "npm run build",
+    "prepare": "husky install && npm run build",
```

Can you make this change while you're at it?
```suggestion
    "prepare": "husky install && yarn build",
```